assets/scss/modules/_side-navigation.scss

```diff
@@ -52,7 +88,112 @@
       color: $govuk-blue;
       @include bold-16;
       background-color: transparent;
-
     }
   }
 }
+
+/*
+Two-level
+
+Two-level navigation shows a submenu to the right of the primary menu.
+
+Markup:
+<div class="side-nav">
+  <ul class="side-nav__component side-nav__quarter-width" id="main-menu">
```

Comment:
if the `id` are not needed here please remove. We need to keep comp lib markup as lean and concise as possible